### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by aeh1707**
- [x] Diagram a data model clearly and explain what it's doing.
- [x] Create SQL queries to build it.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+);
+
+CREATE TABLE invoice_items (
+  id BIGSERIAL PRIMARY KEY,
+  unit_price DECIMAL NOT NULL,
+  quantity INT NOT NULL,
+  total_price DECIMAL NOT NULL,
+  invoice_id INT REFERENCES invoices(id) NOT NULL,
+  treatement_id INT REFERENCES treatments(id)
+);
```
Comment (ShahierNashaat) [inline schema_based_on_diagram.sql:40]: - [ ] Kindly notice that all "join table" for the many-to-many relationships is created, even if they are not shown on the diagram. Therefore you should create a table between `medical_histories` and `treatments`.
```sql
CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

- [ ] Also notice that all FOREIGN KEYS have indexes created.
* `CREATE INDEX ON medical_histories (patient_id);`
* `CREATE INDEX ON invoices (medical_history_id);`
* `CREATE INDEX ON invoice_items (invoice_id);`
* `CREATE INDEX ON invoice_items (treatment_id);`
* `CREATE INDEX ON medical_histories_has_treatments (medical_history_id);`
* `CREATE INDEX ON medical_histories_has_treatments (treatment_id);`

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @DuaneDave, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
- [ ] Kindly notice that the README file should be like that Microverse [Template](https://github.com/microverseinc/readme-template/blob/master/README.md).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (DuaneDave) [reply schema_based_on_diagram.sql:40]: Hey @ShahierNashaat 👋 

Thank you for the review, but creating indexes on the tables wasn't a requirement for this project.
Please review the requirements and as you can also see those tables are already indexed with a constraint on their ID

**Review by prolajumokeoni — APPROVED** [review]
Hi @DuaneDave and @aeh1707 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀  
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (DuaneDave) [reply schema_based_on_diagram.sql:40]: Thank you @prolajumokeoni 😄

**Pull request merged**